web/package.json

```diff
@@ -27,6 +30,7 @@
     "keycloak-js": "^9.0.3",
     "prop-types": "^15.7.2",
     "react": "^16.13.1",
+    "react-bootstrap": "^2.5.0",
```

Comment:
We're importing the minified distribution code from the `bootstrap` package in `web/src/index.tsx`, but we don't actually import `react-bootstrap` anywhere. `react-bootstrap` should be left out unless we're using the components from that library directly, I think.